### PR TITLE
882-fe-sso-create-table-dont-include-checkbox-and-icon-for-now

### LINKED ISSUE #882: [FE][SSO] Create table, don't include checkbox and icon for now
Screenshot 2024-01-25 at 09 03 28

### PR DESCRIPTION
**Pull request by marmorrei**

### CODE REVIEW

File: sso_front/src/components/organisms/UsersTable.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        </td>
+        <td>{user.registrationDate}</td>
+        <td className="button-container">
+          <ButtonStyled size="small" outline>
+            {buttonTxt}
+          </ButtonStyled>
+        </td>
+      </tr>
+    )
+  })
```
Comment (NataliaR-BCN) [inline sso_front/src/components/organisms/UsersTable.tsx:108]: You should use styled-components

File: sso_front/src/components/organisms/UsersTable.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          <th>{t('Especialización')}</th>
+          <th>
+            {t('Estado')}
+            <SortDownIcon src={icons.sortDown} alt="sort-down" />
+          </th>
+          <th>{t('Fecha alta')}</th>
+          <th>{t('Acciones')}</th>
+        </tr>
+      </thead>
+      <tbody>{userRows}</tbody>
```
Comment (NataliaR-BCN) [inline sso_front/src/components/organisms/UsersTable.tsx:124]: userRows should be a component, maybe a molecule

File: sso_front/src/pages/Home.tsx [hunk 3]
```diff
@@ -25,13 +30,44 @@ const MainDiv = styled(FlexBox)`
...
+export type TStatus = 'pendiente' | 'activo' | 'bloqueado'
+
+export type TUser = {
+  name: string
+  specialization: string
+  status: TStatus
+  registrationDate: string
+}
+
+const mockUsers: TUser[] = [
```
Comment (kevinmamaqi) [inline sso_front/src/pages/Home.tsx:42]: Eliminar

File: sso_front/src/pages/Home.tsx [hunk 4]
```diff
@@ -25,13 +30,44 @@ const MainDiv = styled(FlexBox)`
   color: ${colors.gray.gray3};
 `
 
+export type TStatus = 'pendiente' | 'activo' | 'bloqueado'
+
+export type TUser = {
```
Comment (kevinmamaqi) [inline sso_front/src/pages/Home.tsx:35]: Eliminar

File: sso_front/src/pages/Home.tsx [hunk 5]
```diff
@@ -25,13 +30,44 @@ const MainDiv = styled(FlexBox)`
   color: ${colors.gray.gray3};
 `
 
+export type TStatus = 'pendiente' | 'activo' | 'bloqueado'
```
Comment (kevinmamaqi) [inline sso_front/src/pages/Home.tsx:33]: Eliminar

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

**Comment by marmorrei** [comment]
Due tu restructuring of monorepo

**Pull request closed without merging**